const deleteInDB=async ()=>{
    try{
        await Product.deleteOne({
            name:'max8'
        })
        console.log(data);
    }
    catch(err){
        console.log(err);
        
    }
}
deleteInDB();
